packages/react-components/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -24,7 +27,6 @@ const useRootStyles = makeStyles({
 
     ...shorthands.overflow('hidden'),
     textOverflow: 'ellipsis',
```

Comment:
Will text ever overflow now that it wraps?

Reply:
Good catch! I'll remove the style.